.jenkins/common_job_properties.groovy

```diff
@@ -171,6 +180,20 @@ class common_job_properties {
     setPullRequestBuildTrigger(context, commitStatusName, successComment)
   }
 
+  // Enable triggering postcommit runs against pull requests. Users can comment the trigger phrase
+  // specified in the postcommit job and have the job run against their PR to run
+  // tests not in the presubmit suite for additional confidence.
+  static def enablePhraseTriggeringFromPullRequest(def context,
```

Comment:
I like having them have to call it if they want it though; it means that you can tell at a glance in the job config file what's going on. I don't think it's that much more boilerplate, and if we don't have this method we have to put the check_title and trigger_phrase args inside setPostCommit, as well as don't_build_prs. It's going to become unwieldy. I think this is better.